draft-dss-star.md

```diff
@@ -101,200 +101,567 @@ In this document we describe one such system, namely Distributed Secret Sharing
 The following terms are used:
 
 Aggregation Server:
-: An entity that provides some tool/software, that would like to learn aggregated data points from their user-base.
+: An entity that would like to learn aggregated data from users.
 
 Randomness Server:
-: An entity that runs an oblivious pseudorandom function ({{!OPRF=I-D.irtf-cfrg-voprf}}) service that allows clients to receive pseudorandom function evaluations on their measurement and the server OPRF key, without the Randomness Server learning anything about their measurement. The clients use the output as randomness to produce the message that is then sent to the Aggregation Server.
+: An entity that runs an oblivious pseudorandom function ({{!OPRF=I-D.irtf-cfrg-voprf}}) service that allows clients to receive pseudorandom function evaluations on their measurement and the server OPRF key, without the Randomness Server learning anything about their measurement. The clients use the output as randomness to produce the report that is then sent to the Aggregation Server.
+
+Anonymizing Server:
+: An entity that clients use to decouple their identity (IP address) from their messages sent to the Aggregation Server.
 
 Client:
-: The entity that uses the tool.
+: The entity that provides user data to the system.
 
 Measurement:
-: The unencrypted, potentially-sensitive data point that the client is asked to report.
+: The unencrypted, potentially-sensitive data that the client is asked to report.
 
-Message:
+Report:
 : The encrypted measurement being sent by the client.
 
 Auxiliary Data:
-: Arbitrary data that clients may send as part of their message, but which is only revealed when at least K encrypted measurements of the same value are received.
+: Arbitrary data that clients may send as part of their report, but which is only revealed when at least K encrypted measurements of the same value are received.
+
+REPORT_THRESHOLD:
+: The minimum number of reports that an Aggregation Server needs before revealing client data. This value is chosen by the application.
+
+# Cryptographic Dependencies
+
+STAR depends on the following cryptographic protocols and primitives:
```

Comment:
This looks good to me - is it common to have this kind of section in protocol documents that deal with crypto primitives?